website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,357 @@
+    <Doc::Placeholder @height="40px" @text="Item 2" @background="#e5ffd2" />
+    <Doc::Placeholder @height="40px" @text="Item 3" @background="#d2f4ff" />
+    <Doc::Placeholder @height="40px" @text="Item 4" @background="#fff8d2" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### Align
+
+Use the `@align` argument to align grid items to the "start", "end", "center" or "stretch" them within the grid parent.
+
+Note: The `Grid` parent will need a height set for the effect to be visible.
+
+```handlebars
+<div class="doc-grid-mobile-view">
+  <Hds::Layout::Grid @columnMinWidth="50px" @gap="16" @align="center" {{style height="100%"}}>
+    <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4c5f3" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e5ffd2" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#d2f4ff" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#fff8d2" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### Colspan & rowspan
+
+Use the `colspan` and `rowspan` arguments of the `Grid::Item` component to set the number of columns or rows an item should occupy.
+
+The following example has an underlying 4-column grid specified by setting a `columnMinWidth` of “25%”. It uses `colspan` and `rowspan` to create a flexible layout roughly resembling a typical web page layout.
+
+Note: By default, if a height is set on the `Grid` parent, grid row heights will stretch proportionally to fill the `Grid`. To instead make a row conform to the minimum height of its content, you can pass an inline style as shown in the example.
+
+```handlebars
+<div {{style height="400px" border="1px solid"}}>
+  <Hds::Layout::Grid 
+    @columnMinWidth="25%" 
+    @gap="12"
+    {{style height="100%" grid-template-rows="min-content"}}
+    as |LG|
+  >
+    <LG.Item @colspan={{4}}>
+      <Doc::Placeholder @text="Item 1" @background="#e4c5f3" {{style padding="1em"}} />
+    </LG.Item>
+
+    <LG.Item @rowspan={{3}}>
+      <Doc::Placeholder @height="100%" @text="Item 2" @background="#e5ffd2" />
+    </LG.Item>
+
+    <LG.Item @colspan={{3}}>
+      <Doc::Placeholder @height="100%" @text="Item 3" @background="#d2f4ff" />
+    </LG.Item>
+
+    <LG.Item @colspan={{3}} @rowspan={{2}}>
+      <Doc::Placeholder @height="100%" @text="Item 4" @background="#fff8d2" />
+    </LG.Item>
+  </Hds::Layout::Grid>
+</div>
+```
+
+---
+
+## Common layout patterns
+
+Below are examples of common layout patterns that can be achieved using the `Layout::Grid` component in combination with other HDS components.
+
+!!! Warning
+
+**Important**
+
+The examples below are meant to show how one _could_ use the `Layout::Grid` component to implement certain common/standard UI patterns. They're **not** meant to be taken literally as they are and be used in production code. Also, some of these patterns may already be implemented directly in HDS components that are ready to use.
```

Comment:
```suggestion
The examples below are meant to show how one _could_ use the `Layout::Grid` component to implement certain common/standard UI patterns. They're **not** meant to be taken literally as they are and be used in production code. 
```

Suggestion: this line was taken out from the flex docs because its not really necessary to say